spec/system/admin/orders_spec.rb

```diff
@@ -65,60 +65,69 @@
 
-      page.find('.filter-actions .button.icon-search').click
+        # Order 2 and 3 should show, but not 4
+        expect(page).to have_content order2.number
+        expect(page).to have_content order3.number
+        expect(page).to_not have_content order4.number
+      end
 
-      # Order 2 and 3 should show, but not 4
-      expect(page).to have_content order2.number
-      expect(page).to have_content order3.number
-      expect(page).to_not have_content order4.number
-    end
+      it "filter by distributors" do
+        select2_select distributor2.name.to_s, from: 'q_distributor_id_in'
+        select2_select distributor4.name.to_s, from: 'q_distributor_id_in'
 
-    it "filter by distributors" do
-      login_as_admin_and_visit 'admin/orders'
+        page.find('.filter-actions .button.icon-search').click
 
-      select2_select distributor2.name.to_s, from: 'q_distributor_id_in'
-      select2_select distributor4.name.to_s, from: 'q_distributor_id_in'
+        # Order 2 and 4 should show, but not 3
+        expect(page).to have_content order2.number
+        expect(page).to_not have_content order3.number
+        expect(page).to have_content order4.number
+      end
 
-      page.find('.filter-actions .button.icon-search').click
+      it "filter by complete date" do
+        find("input.datepicker").click
+        select_dates_from_daterangepicker(order3.completed_at.yesterday, order4.completed_at.tomorrow)
 
-      # Order 2 and 4 should show, but not 3
-      expect(page).to have_content order2.number
-      expect(page).to_not have_content order3.number
-      expect(page).to have_content order4.number
-    end
+        page.find('.filter-actions .button.icon-search').click
 
-    it "filter by complete date" do
-      login_as_admin_and_visit 'admin/orders'
+        # Order 3 and 4 should show, but not 2
+        expect(page).to_not have_content order2.number
+        expect(page).to have_content order3.number
+        expect(page).to have_content order4.number
+      end
 
-      find("input.datepicker").click
-      select_dates_from_daterangepicker(order3.completed_at.yesterday, order4.completed_at.tomorrow)
+      it "filter by email" do
+        fill_in "Email", with: customer3.email
 
-      page.find('.filter-actions .button.icon-search').click
+        page.find('.filter-actions .button.icon-search').click
 
-      # Order 3 and 4 should show, but not 2
-      expect(page).to_not have_content order2.number
-      expect(page).to have_content order3.number
-      expect(page).to have_content order4.number
+        # Order 3 should show, but not 2 and 4
+        expect(page).to_not have_content order2.number
+        expect(page).to have_content order3.number
+        expect(page).to_not have_content order4.number
+      end
     end
 
     context "select/unselect all orders" do
       before do
-        login_as_admin_and_visit spree.admin_orders_path
+        login_as_admin_and_visit 'admin/orders'
```

Comment:
Why so?

Reply:
I've experimented with it and then forgot to take it back, I actually wanted to ask what the difference here would be. Is any reason to favor one of the options of the other one?

Reply:
I guess using `spree.admin_orders_path` is more resilient if one day we want to change the routing of that path or the way we want to routes admin routes for example.

Reply:
I see, that makes sense - thanks JB :+1:

Reply:
Programmers tend to avoid plain strings because they don't have any meaning and you often find out late if they are wrong or outdated. A variable or method name like `admin_orders_path` will raise an error if you spelled it wrong. And when you refactor the code and rename it then a clever editor can do that for you.